mixes/reducer.js

```diff
@@ -101,18 +104,18 @@ function createReducer (config) {
     ])),
     [deleteMixSuccess]: (state, action) => {
       const nestedMix = action.payload
+      const { id, channel } = nestedMix
 
       const nextRecords = { ...state.records }
-      delete nextRecords[nestedMix.id]
-
-      // TODO: we need to do the inverse of setChannels, setClips. how?
+      delete nextRecords[id]
 
       return loop({
         ...state,
-        dirty: without(state.dirty, nestedMix.id),
+        dirty: without(state.dirty, id),
         records: nextRecords
       }, Effects.batch([
-        Effects.constant(deleteMeta(nestedMix.id)),
+        Effects.constant(unsetChannel(channel)),
```

Comment:
**update 1/29**: i changed `unsetChannel` and `deleteMix` to take an `id`. this made a lot more sense to parallel cases like `undirtyClip` etc.

whether or not i change `saveMix` to take an id is up for the future